workspaces/adoption-insights/package.json

```diff
@@ -51,7 +51,13 @@
   },
   "resolutions": {
     "@types/react": "^18",
-    "@types/react-dom": "^18"
+    "@types/react-dom": "^18",
```

Comment:
Why do you need resolutions for backstage dependencies here ?

Reply:
I am hitting this issue - https://github.com/backstage/backstage/issues/28925, so I had to add the resolutions or the backend wont start.

Reply:
Maybe having the resolution on `@backstage/plugin-scaffolder` only would be sufficient ?

Reply:
added `@backstage/plugin-scaffolder-backend` and `@backstage/plugin-catalog-backend` and removed all others.